index.html

```diff
@@ -0,0 +1,144 @@
+        </div>
+      </div>
+    </header>
+
+    <section class="scills">
+      <h3 class="scills__text">Objective & Summary</h3>
+      <ul class="scills list">
+        <li class="list__item">
+          Start a career in EPAM and in 3 years to grow to team lead
+        </li>
+      </ul>
+
+      <h3 class="scills__text">Key skills:</h3>
+
+      <ul class="scills list">
+        <li class="list__item">
+          Programming languages: Java Script – basics
+        </li>
+        <li class="list__item">Development Tools: Visual Studio Code</li>
+        <li class="list__item">
+          Web Technologies: HTML\СSS\Java Script – basics
+        </li>
+        <li>Defect & bug discovery</li>
+      </ul>
+
+      <h3 class="scills__text">Business qualities:</h3>
+
+      <ul class="scills list">
+        <li class="list__item">Able to manage a team</li>
+        <li class="list__item">Sociable</li>
+        <li class="list__item">Resourceful</li>
+        <li class="list__item">Responsible</li>
+        <li class="list__item">Purposeful</li>
+        <li class="list__item">Punctual</li>
+        <li class="list__item">
+          Non-confrontational, but able to defend my own point of view
+        </li>
+        <li class="list__item">Fast learner</li>
+        <li class="list__item">
+          Non-confrontational, but able to defend my own point of view
+        </li>
+      </ul>
+
+      <h3 class="scills__text">Education</h3>
+
+      <ul class="scills list">
+        <li class="list__item">
+          Higher education (2005-2010). Belarusian National Technical University
+          Transport communication faculty Handling, construction, road-building
+          machinery and equipment Engineer
+        </li>
+      </ul>
+
+      <h3 class="scills__text">Experience</h3>
+
+      <ol class="scills list">
+        <li class="list__item">
+          Engineer(08.2010-11.2013).The Belarusian Optical and Mechanical
+          Association- the Minsk Mechanical Works named after S.I. Vavilov
+          (Minsk), optomechanical and optoelectronic equipment Engineer.
+        </li>
+        <li class="list__item">
+          Chief engineer(08.2013-2018).The Belarusian Optical and Mechanical
+          Association- the Minsk Mechanical Works currently named after S.I.
+          Vavilov (Minsk), optomechanical and optoelectronic equipment working
+          Chief engineer
+        </li>
+        <li class="list__item">
+          Chief engineer(2013-2018).The Belarusian Optical and Mechanical
+          Association- the Minsk Mechanical Works currently named after S.I.
+          Vavilov (Minsk), optomechanical and optoelectronic equipment working
+          Chief engineer
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="https://github.com/EvgenPrushk"
+            >2018 Learning rolling-scopes-school (github.com/EvgenPrushk)
+          </a>
+        </li>
+      </ol>
+

```

Comment:
do not use line breaks for html text. this can lead to some problems in different browsers